sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/Azure.ResourceManager.DnsResolver.Tests.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.ResourceManager.DnsResolver.csproj" />
+    <ProjectReference Include="..\..\..\network\Azure.ResourceManager.Network\src\Azure.ResourceManager.Network.csproj" />
```

Comment:
Please use the package reference instead of the project reference, we have released the stable version for [Azure.ResourceManager.Network](https://www.nuget.org/packages/Azure.ResourceManager.Network).